charts/celestia-node/templates/statefulset.yaml

```diff
@@ -272,6 +276,16 @@ spec:
             - name: home-dir # This is needed so that the process has permissions to create files in the home directory
               mountPath: {{ .Values.node.settings.home }}
               readOnly: false
+            {{- if .Values.node.settings.wssTLS.enabled }}
+            - name: tls-certs
+              mountPath: {{ .Values.node.settings.home }}/tls/cert.pem
+              subPath: fullchain.pem
+              readOnly: true
+            - name: tls-certs
+              mountPath: {{ .Values.node.settings.home }}/tls/key.pem
+              subPath: privkey.pem
+              readOnly: true
+            {{- end }}
```

Comment:
When using cert-manager, the keys are named `tls.cert` and `tls.key`.
And we need to mount them as `cert.pem` and `key.pem`.